conda/recipes/meta.yaml

```diff
@@ -18,17 +16,18 @@ source:
 
 build:
   number: {{ git_revision_count }}
+  noarch: python
```

Comment:
This indicates that we have a pure Python package that can run on any Python version.